### PR TITLE
Letterhead in user progress

### PR DESCRIPTION
**Pull request by shreyashah115**
#11163 
letterheadd

### CODE REVIEW

File: erpnext/utilities/user_progress.py [hunk 1]
```diff
@@ -81,6 +81,26 @@ def get_slide_settings():
...
+
+		frappe._dict(
+			action_name='Add Letterhead',
+			domains=('Manufacturing', 'Services', 'Retail', 'Distribution', 'Education'),
+			title=_("Add Letterhead"),
+			help=_("Upload your letter head. (you can edit it later)."),
+			fields=[
+				{"fieldtype":"Attach Image", "fieldname":"letterhead",
+					"label":_("Letterhead"),
+					"description": _("Keep it web friendly 900px (w) by 100px (h)"),
```
Comment (pratu16x7) [inline erpnext/utilities/user_progress.py]: Let's just skip the label and description altogether. Will also help in keeping the image control in the centre. Instead, we'll change help to "Upload your letter head (Keep it web friendly as 900px by 100px)". Once inside, they'll already be aware that things can be changed later on. (The edit line was while in the setup wizard).

**Review by pratu16x7 — CHANGES_REQUESTED** [review]

**Comment by pratu16x7** [comment]
Also update `install_fixtures.py`.

**Comment by rmehta** [comment]
Also update Setup Wizard test!

**Comment by pratu16x7** [comment]
@rmehta Agree, if we are also removing the brand info in this PR (right now there are no changes in Setup Wizard)

**Comment by nabinhait** [comment]
@shreyashah115 Merged this PR. Remove brand slide from setup-wizard via a separate PR.

**Pull request merged**